iBudget/fund/views.py

```diff
@@ -21,3 +24,42 @@ def show_fund(request):
             user_funds.append({'id': entry.id, 'name': entry.name})
         return JsonResponse(user_funds, status=200, safe=False)
     return JsonResponse({}, status=400)
+
+
+@require_http_methods(["POST"])
+def create_new_fund(request):
```

Comment:
Please take a look how you could optimise a code for this method.
Also all this changes should be made at once or none of them at all.

Reply:
l will think about it